gomint-server/src/main/java/io/gomint/server/plugin/SimplePluginManager.java

```diff
@@ -647,6 +647,11 @@ public String getBaseDirectory() {
         return (T) this.installedPlugins.get(name);
     }
 
+    @Override
+    public Map< String, Plugin > getPlugins() {
+        return this.installedPlugins;
```

Comment:
Please return an immutable map.

`ImmutableMap.copyOf(this.installedPlugins)`

Reply:
oops my bad